src/main/java/MainOptions.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+
+public class MainOptions {
+    Scanner scan = new Scanner(System.in);
+    public void mainOptions(){
```

Comment:
Esse método está imprimindo o menu, o que acha de deixar esse nome mais explicito?

```suggestion
    public void showMenuOptions(){
```